### ISSUE TITLE
Confusing about the initial parameter problem.

### ISSUE THREAD

**Issue by zhyliu00** [princetonvisualai/RememberThePast-DatasetDistillation#4]
Hi.

This is an inspiring work about dataset distillation. The training pipeline (Algorithm 1 in the paper) aims to maximize the performance of the model that 

- trained by synthetic data D_s  (Algorithm lines 8-13)
- evaluated by real data D_{tr} (Algorithm line 14).

This pipeline is similar to Dataset Distilation[1], which also aims to maximize the performance trained by S and evaluated by T. However, [1] raises a problem that the **initial parameter $\theta_0$** (Algorithm 1 line 6) significantly influences the performance of the whole algorithm. This is because the final parameter $\theta_T$ is dependent on the initial parameter $\theta_0$ and thus the loss function $J(\phi)$ and optimization direction **OPT-STEP $(\phi, J(\phi), \alpha_1, \beta_1)$** is dependent on the initial parameter $\theta_0$.

So, I'm confusing how this work deals with this initial parameter problem.

Thanks!~

[1]Wang, T., Zhu, J., Torralba, A., & Efros, A. A. (2018). Dataset Distillation. ArXiv. https://doi.org/10.48550/arXiv.1811.10959

**Comment by Lucas2012** [comment]
Hi,

Thanks for your interest in our work. In our algorithm, every training step will randomly sample (initialize) a new set of network weights and train from scratch using the Synthetic data (this is also true in several prior works/methods, e.g. Gradient Matching) to compute generalization loss. This random initialization can lead to a set of synthetic images that are generalizable across initial parameters or even cross architectures (e.g. distill using ConvNet and re-train using ResNet or AlexNet).

**Comment by zhyliu00** [comment]
Thanks for your reply!
